lib/tlsca/ca.go

```diff
@@ -190,6 +190,9 @@ type Identity struct {
 
 	// DeviceExtensions holds device-aware extensions for the identity.
 	DeviceExtensions DeviceExtensions
+
+	// IsSSOUser indicates that this identity is from an SSO user.
+	IsSSOUser bool
```

Comment:
just to consider if the bool variable can be replaced by a enum like: UserType in case of future extensibility.

Reply:
Added the enum to the Identity :+1: 
Event is still consuming the boolean value